ci/script_linux.sh

```diff
@@ -1,7 +1,7 @@
 make
 if [ $COVERALLS = ON ]; then make coveralls; fi
 sudo ldconfig --verbose # So the test executeables can detect libtinyxml2
-if [ $BUILD_CORE_ONLY = OFF ]; then make test; fi
+if [ $BUILD_CORE_ONLY = OFF ]; then make all test tests; fi
```

Comment:
> This raises another important question: should we also build tutorials and examples here, to verify that they build successfully?

I would prefer to do complete build test in travis unless it exceeds the build time limit.